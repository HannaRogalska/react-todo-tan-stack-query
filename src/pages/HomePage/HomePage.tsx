

const HomePage = () => {
  return (
    <div className="flex flex-col items-center">
      <h1 className="text-4xl font-bold mb-4">
        Organize Your Day
      </h1>
      <p className=" mb-8 max-w-xl">
        Create, manage, and complete your tasks easily. Stay productive and keep
        track of everything in one place.
      </p>
    </div>
  );
}

export default HomePage
