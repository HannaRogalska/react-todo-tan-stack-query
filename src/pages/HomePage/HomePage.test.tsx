import { render, screen } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import HomePage from "./HomePage";

describe("HomePage", () => {
  it("render text", () => {
      render(<HomePage />);
    expect(screen.getByText("home")).toBeInTheDocument();
  });
  it("matcher snapshot", () => {
      const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
