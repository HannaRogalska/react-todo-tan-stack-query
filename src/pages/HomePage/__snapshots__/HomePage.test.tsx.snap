// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomePage > matcher snapshot 1`] = `
<div>
  <div
    class="flex flex-col items-center"
  >
    <h1
      class="text-4xl font-bold mb-4"
    >
      Organize Your Day
    </h1>
    <p
      class=" mb-8 max-w-xl"
    >
      Create, manage, and complete your tasks easily. Stay productive and keep track of everything in one place.
    </p>
  </div>
</div>
`;
